app/views/partnering_organizations/registrations/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= devise_error_messages! %>
+
+ <div><%= f.label :name %><br />
+        <%= f.text_field :name, autofocus: true %></div>
+
+ <div><%= f.label :website %><br />
+        <%= f.text_field :website, autofocus: true %></div>
+
+<div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true %>
+  </div>
+  
+
+  <div>
+  <%= f.label :role %><br />
+  <%= f.select :role, options_for_select(%w[Select Care Giver Support Team Educator Volunteer Organization Awareness Team]) %>
```

Comment:
these options should pull from the enum defined in the partnering_organization model